com.unity.ml-agents/Editor/BehaviorParametersEditor.cs

```diff
@@ -2,14 +2,14 @@
 using UnityEditor;
 using Barracuda;
 
-namespace MLAgents
+namespace MLAgents.Editor
 {
     /*
      This code is meant to modify the behavior of the inspector on Agent Components.
     */
     [CustomEditor(typeof(BehaviorParameters))]
     [CanEditMultipleObjects]
-    internal class BehaviorParametersEditor : Editor
+    internal class BehaviorParametersEditor : UnityEditor.Editor
```

Comment:
did this not compile because of a namespace conflict?

Reply:
Editor could be either UntiyEditor.Editor or MLAgents.Editor. So I am disambiguating...